virtualmachine/vsphere/util.go

```diff
@@ -429,10 +429,21 @@ var cloneFromTemplate = func(vm *VM, dcMo *mo.Datacenter, usableDatastores []str
 		Datastore: &dsMor,
 	}
 
+        hotAddMemory := true
```

Comment:
You can either move it to top as global variables.
Or set it directly in below initialization of config.